data/isdv4-51af.tablet

```diff
@@ -0,0 +1,15 @@
+# this is for the Wacom pen + touchscreen as found in the Lenovo ThinkPad X390 Yoga
+
+[Device]
+Name=Wacom ISDv4 51af
+ModelName=
+DeviceMatch=i2c:056a:51af
```

Comment:
Is this correct?
I added this file in `/usr/share/libwacom/`, and `libwacom-list-local-devices` still showed nothing until I changed it from `i2c:` to `usb:`
```# Device node: /dev/input/event22
[Device]
Name=Wacom ISDv4 51af
ModelName=
DeviceMatch=usb:056a:51af;
Class=ISDV4
Width=12
Height=7
IntegratedIn=Display;System;
Styli=0xffffe;0xfffff;

[Features]
Reversible=false
Stylus=true
Ring=false
Ring2=false
Touch=true
TouchSwitch=false
# StatusLEDs=
NumStrips=0
Buttons=0

---------------------------------------------------------------
# Device node: /dev/input/event19
[Device]
Name=Wacom ISDv4 51af
ModelName=
DeviceMatch=usb:056a:51af;
Class=ISDV4
Width=12
Height=7
IntegratedIn=Display;System;
Styli=0xffffe;0xfffff;

[Features]
Reversible=false
Stylus=true
Ring=false
Ring2=false
Touch=true
TouchSwitch=false
# StatusLEDs=
NumStrips=0
Buttons=0

---------------------------------------------------------------```

Reply:
*DOH!* I made a mental note to swap `i2c` to `usb` in the file I copied from, but I must have got distracted and forgot. Change incoming.